app/controllers/sql-lite-service.js

```diff
@@ -75,30 +76,30 @@ module.exports = function (app) {
     /**
      * tables
      */
-    function createWalletTokens() {
-        return new Promise((resolve, reject) => {
-            knex.schema.hasTable('wallet_tokens').then(function (exists) {
-                if (!exists) {
-                    knex.schema.createTable('wallet_tokens', (table) => {
-                        table.increments('id');
-                        table.integer('walletId').notNullable().references('wallets.id');
-                        table.integer('tokenId').notNullable().references('tokens.id');
-                        table.decimal('balance').defaultTo(0);
-                        table.integer('recordState').defaultTo(1);
-                        table.integer('createdAt').notNullable().defaultTo(new Date().getTime());
-                        table.integer('updatedAt');
-                    }).then((resp) => {
-                        console.log("Table:", "wallet_tokens", "created.");
-                        resolve("wallet_tokens created");
-                    }).catch((error) => {
-                        reject(error);
-                    });
-                } else {
-                    resolve();
-                }
-            });
-        });
-    }
+    // function createWalletTokens() {
```

Comment:
Can this be deleted?

Reply:
yeah it is replicated in the seed file along with the others

Reply:
yes can delete this, its been replicated in the seed file now